orderbook/src/database/orders.rs

```diff
@@ -201,8 +201,51 @@ struct OrdersQueryRow {
     signing_scheme: DbSigningScheme,
 }
 
+/// Amount which is the minimum to consider an order fully filled.
+/// original amount / FILLED_PERCENTAGE_DENOMINATOR => minimum amount
+/// Fully filled order => amount - executed_amount < minimum_amount
+/// The higher this number, the smaller the amount to consider an order filled.
+const FILLED_PERCENTAGE_DENOMINATOR: u32 = 10_000; // 10_000 == 0.01 %
```

Comment:
Don't fill like this is the right place for this. Anywhere better to put it?

Reply:
I'm not a fan of having the backend be opinionated of when an order should be considered fully filled (even when it's not). If the order was in a "final" state I would expect it to be impossible to match it further (we might even filter solvable orders based on that). However, for very large orders even a small percentage may still be fillable, e.g. when price moves significantly, so I find it problematic to hard code a FILLED_PERCENTAGE_DENOMINATOR and would rather only treat orders as fully filled when they actually are.

Reply:
Fair enough, will revert the change.